camera.go

```diff
@@ -20,6 +20,65 @@ type CameraTerminal struct {
 	CameraDescriptor *descriptors.CameraTerminalDescriptor
 }
 
+func (ct *CameraTerminal) GetAutoExposureMode() (descriptors.AutoExposureMode, error) {
+	ifnum := ct.usb.altsetting.bInterfaceNumber
+
+	fac := &descriptors.AutoExposureModeControl{}
+	size := fac.MarshalSize()
+
+	buf := C.malloc(C.ulong(size))
+	defer C.free(buf)
+
+	if ret := C.libusb_control_transfer(
+		ct.deviceHandle,
+		C.uint8_t(requests.RequestTypeVideoInterfaceGetRequest),                                     /* bmRequestType */
+		C.uint8_t(requests.RequestCodeGetCur),                                                       /* bRequest*/
+		CameraTerminalControlSelectorAutoExposureModeControl<<8,                                     /* wValue: CT_AE_MODE_CONTROL on the hight byte */
+		C.uint16_t(uint16(ct.CameraDescriptor.InputTerminalDescriptor.TerminalID)<<8|uint16(ifnum)), /* wIndex*/
+		(*C.uchar)(buf),  /* data */
+		C.uint16_t(size), /* len */
+		0,                /* timeout */
+	); ret < 0 {
+		return -1, fmt.Errorf("libusb_control_transfer failed: %w", libusberror(ret))
```

Comment:
```suggestion
		return 0, fmt.Errorf("libusb_control_transfer failed: %w", libusberror(ret))
```

Return 0 in failure cases.